src/Model/Contact.php

```diff
@@ -2663,6 +2660,15 @@ public static function addRelationship(array $importer, array $contact, array $d
 
 			Post\UserNotification::insertNotification($contact['id'], Activity::FOLLOW, $importer['uid']);
 
+			DI::notify()->createFromArray([
```

Comment:
The `notification` table eventually should completely replace the `notify` table, also the `notify` table is updated via the function above which then calls this function: https://github.com/friendica/friendica/blob/develop/src/Model/Subscription.php#L157